lib/spaces/providers/packing/docker/docker.rb

```diff
@@ -5,7 +5,10 @@ module Providers
   class Docker < ::ProviderAspects::Provider
     extend Docker
 
-    alias_method :pack, :emission
+    ::Docker.options[:read_timeout] = 1000
```

Comment:
at this place in the code, I think these these could just be `options[:read_timeout] = 1000` ... happy to check it out later